src/components/HomeSlides.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Carousel } from "3d-react-carousal";
+import "./style.css";
+const HomeSlides = () => {
+  // TODO: get the books images as a prop from the parent (Home)
```

Comment:
Why not set it now? Have an array of objects be passed from the parent and loop through it here.